packages/documentation/.storybook/blocks/footer.tsx

```diff
@@ -29,59 +29,69 @@ const DEVELOPERS: Developer[] = [
   },
 ];
 
-export default () => (
-  <footer className="docs-footer mt-huge bg-light">
-    <div className="container">
-      <div className="pt-big-r pb-big-r">
-        <h2 className="mt-0">Support</h2>
+const BASEURL = 'https://github.com/swisspost/design-system/tree/main/packages/documentation';
 
-        <div className="d-flex flex-wrap mt-huge-r mb-huge-r profile-list">
-          {DEVELOPERS.sort(() => (Math.random() > 0.5 ? 1 : -1)).map((developer, index) => (
-            <article key={index} className="avatar">
-              <img
-                className="profile-picture"
-                src={developer.avatar}
-                alt={`Profile picture ${developer.name}`}
-              />
-              <div>
-                <p>
-                  <strong>{developer.name}</strong>
-                </p>
-                <p>{developer.title}</p>
-              </div>
-            </article>
-          ))}
-        </div>
+function getGitHubUrl(path: String) {
+  return `${BASEURL}${path.replace(/^\./, '').replace(/\.stories\.ts$/, '.docs.mdx')}`;
+}
 
-        <div className="row mt-regular-r">
-          <div className="col-12 col-rg-auto mt-regular-r">
-            <a
-              className="btn-primary btn btn-rg btn-animated"
-              href="https://github.com/swisspost/design-system/issues"
-              target="_blank"
-              rel="noopener"
-            >
-              <span>Submit an issue</span>
-            </a>
+export default (params: { pathToStoryFile: String }) => (
+  <div>
```

Comment:
As this `div` is not necessary (just there to wrap the content), you can then use [a fragment](https://react.dev/reference/react/Fragment). The difference is it is not rendered in the page in the end.
```suggestion
  <>
```